src/shared/utilities/vsCodeUtils.ts

```diff
@@ -162,3 +163,19 @@ export function reloadWindowPrompt(message: string): void {
         }
     })
 }
+
+/**
+ * Opens a URL in the system web browser. Throws `CancellationError`
+ * if user dismisses the vscode confirmation prompt.
+ */
+export async function openUrl(url: vscode.Uri): Promise<boolean> {
+    return telemetry.aws_openUrl.run(async span => {
+        span.record({ url: url.toString() })
+        const didOpen = await vscode.env.openExternal(url)
+        if (!didOpen) {
+            throw new CancellationError('user')
+            // getLogger().verbose('failed to open URL: %s', e)
```

Comment:
`openExternal` returning false is ambiguous :(
Always treating it as a cancellation seemed like the most reasonable solution.